esp/xslt/esdl2monitor.xslt

```diff
@@ -154,172 +142,135 @@ MonSoapcall(string requestXml) := FUNCTION
   end;
 
   in_rec Format () := transform
-    Self.<xsl:value-of select="$requestType"/> := request;
+    Self.<xsl:value-of select="$requestType"/> := req;
   end;
 
   ds_request := dataset ([Format()]);
 
-  out_rec SetError (ds_request inn) := TRANSFORM
-    SELF.soap_code := FAILCODE;
-    SELF.soap_message := FAILMESSAGE;
-    self := [];
-  END;
-
   // execute soapcall
   ar_results := SOAPCALL (ds_request,
                           service_url,
                           service_name,
                           {ds_request},
-                          dataset (out_rec),
-                          onFail (SetError (Left)), TIMEOUT(6), RETRY(1));
+                          dataset (the_responseLayout),
+                          TIMEOUT(6), RETRY(1));
```

Comment:
For now I'd rather these be consistent for all monitors... that way I know what to expect.  I could make it part of the template later on.